python/lsst/ctrl/mpexec/cli/opt/show.py

```diff
@@ -0,0 +1,49 @@
+# This file is part of ctrl_mpexec.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import click
+
+from lsst.daf.butler.cli.utils import split_commas
+
+
+class show_option:  # noqa: N801
+
+    defaultHelp = ("Dump various info to standard output. Possible items are: `config', "
```

Comment:
I think this might be better as a `"""` quoted string.

Reply:
@timj as far as I can tell doing it that way puts the newlines and whitespace (if each new line does not begin at column 0) in the value of the string (is there a way to prevent that?). Click does it's own text wrapping for help strings and depends on console width. The extra whitespace messes that up. I guess I could format the `"""` str for narrowest expected console, but I'm not sure that would be a real win over just doing it this way & letting the wookie win.

Reply:
That's disappointing. The help text is very hard to read in that context. The alternative is to use the triple-quoted string and run it through a helper function that replaces every newline with a space.

Reply:
ok, that and `textwrap.dedent` FTW